engine/src/state_chain_observer/client.rs

```diff
@@ -94,6 +96,20 @@ impl DefaultRpcClient {
 	}
 }
 
+#[derive(Error, Debug)]
+pub enum CreateStateChainClientError {
+	#[error("The runtime version '{cfe_version}' is not yet compatible with the current release '{cfe_version_required}'")]
+	NotYetCompatible { cfe_version: SemVer, cfe_version_required: SemVer },
+	#[error("The runtime version '{cfe_version}' is no longer compatible with the current release '{cfe_version_required}' at block {first_incompatible_block}")]
+	NoLongerCompatible {
+		cfe_version: SemVer,
+		cfe_version_required: SemVer,
+		first_incompatible_block: state_chain_runtime::BlockNumber,
+	},
+	#[error(transparent)]
+	Other(#[from] anyhow::Error),
```

Comment:
I thought we weren't going this direction. I think you can just remove this variant.